app/admin/prizes.rb

```diff
@@ -5,30 +5,24 @@
 
   controller do
     def create
-      total_prizes_percentages = Prize.pluck(:percentage).sum
-      if total_prizes_percentages + params[:prize][:percentage].to_i > 100
-        render json: 'Error! The total percentage is greater than 100%!'
-      else
-        Prize.create(prize_params)
-        redirect_to '/admin/prizes'
-      end
+      total_prizes_percentages = Prize.pluck(:percentage).sum + params[:prize][:percentage].to_i
+      total_prizes_percentages > 100 ? (render json: 'Error! The total percentage is greater than 100%!') : super
```

Comment:
What about the 
```
  Prize.create(prize_params)
  redirect_to '/admin/prizes'
```

you had in the `else` case before? Is it the same thing as `super`?

Reply:
Yes, it is. The only difference is that instead of redirecting to the Prizes List is redirecting to the Prize Details. And this way I reduced the code size, so it doesn't complain about code size